### ISSUE TITLE
[error] Sample rate for model (5000) and data (4000) are not compatible.

### ISSUE THREAD

**Issue by AzlanNI** [nanoporetech/dorado#965]
# Issue Report

## Please describe the issue:
I am currently analyzing a MRSA Dataset with 24 different samples. And i want to generate a bam file with information about 6mA, 5mC and 4mC modifications. So i was running the command 
```
for i in {01..03}
do
    barcode_id=barcode${i}
    output=/Users/azlannisar/Snakemake_Output_final/MRSA_Output
    reference_path=/Users/azlannisar/Desktop/mount/MRSA_Data/Reference/$barcode_id/assembly.fasta
    pod5_path=/Users/azlannisar/Desktop/mount/MRSA_Data/Output_MRSA/${barcode_id}

    PYTORCH_CUDA_ALLOC_CONF=max_split_size_mb:25 /Users/azlannisar/Downloads/dorado-0.7.2-osx-arm64/bin/dorado basecaller \
    --modified-bases-models /Users/azlannisar/Downloads/dorado-0.7.2-osx-arm64/lib/dna_r10.4.1_e8.2_400bps_hac@v5.0.0_4mC_5mC@v1,/Users/azlannisar/Downloads/dorado-0.7.2-osx-arm64/lib/dna_r10.4.1_e8.2_400bps_hac@v5.0.0_6mA@v1 \
    /Users/azlannisar/Downloads/dorado-0.7.2-osx-arm64/lib/dna_r10.4.1_e8.2_400bps_hac@v5.0.0 \
    $pod5_path \
    --reference $reference_path > $output/$barcode_id.bam
done
```
This Skript does work for my Metamycoplasma hominis Data. But for my MRSA Data i get the Error :
[error] Sample rate for model (5000) and data (4000) are not compatible.

I looked it up and saw that this is connected with the sampling frequency. But my question is what exactly is the problem over here since i can't use 4khz sampling frequency models these dont have 5mC. 6mA and 4mC modification calling. 

*Please provide a clear and concise description of the issue you are seeing and the result you expect.*

## Steps to reproduce the issue:

*Please list any steps to reproduce the issue.*

## Run environment:
- Dorado version: 0.7.2
- Dorado command: further up
- Operating system: MacOs Sonoma 14.5
- Hardware (CPUs, Memory, GPUs): M1 
- Source data type (e.g., pod5 or fast5 - please note we always recommend converting to pod5 for optimal basecalling performance): pod5
- Source data location (on device or networked drive - NFS, etc.):  on the HPC of the university
- Details about data (flow cell, kit, read lengths, number of reads, total dataset size in MB/GB/TB): “FLO-MIN112” R10 flow cell SQK-NBD112-24 kit
- Dataset to reproduce, if applicable (small subset of data to share as a pod5 to reproduce the issue): i cant upload it here since the data is too big

So basically  the data is sequenced with a  model which had a lower sampling frequency and thats why the mod calls with 5khz models are not working did i get  this right ? And what would be a possible solution. 
## Logs

* Please provide output trace of dorado (run dorado with -v, or -vv on a small subset)

**Comment by StephDC** [comment]
https://github.com/nanoporetech/dorado#dna-models

According to the table, a sample sequenced with 4kHz sampling rate is only compatible with model versions `v4.1.0` or lower. It is also only compatible with `5mCG_5hmCG` modification.

Maybe someone want to train a new model for 4mC, 5mC, 6mA for 4kHz so older data could be reused? Otherwise one may only re-sequence the original sample in 5kHz sampling rate for 4mC, 5mC, 6mA basecalling.

**Comment by HalfPhoton** [comment]
Thank you for the reply @StephDC.

As stated, above the solution is to re-sequence the data with the 5kHz condition so that the data is compatible with the new simplex and modbase models.

Best regards,
Rich
